x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields/custom_fields/global_data_tags_table.test.tsx

```diff
@@ -44,7 +44,7 @@ describe('GlobalDataTagsTable', () => {
         global_data_tags: tags,
       });
 
-      const updateAgentPolicy = React.useCallback((policy) => {
+      const updateAgentPolicy = React.useCallback((policy: any) => {
```

Comment:
Could you leave a comment above this line to remind us to check this type? I took a quick look but there is something strange with the types, it could hide a code smell.
I think it can be fixed on a separate PR.

Reply:
thanks! updating